staging/src/k8s.io/apiserver/pkg/storage/etcd3/metrics/metrics.go

```diff
@@ -47,6 +47,22 @@ var (
 		},
 		[]string{"operation", "type"},
 	)
+	etcdRequestCounts = compbasemetrics.NewCounterVec(
+		&compbasemetrics.CounterOpts{
+			Name:           "etcd_request_total",
```

Comment:
https://prometheus.io/docs/practices/naming/#metric-names
```suggestion
			Name:           "etcd_requests_total",
```

Reply:
Nice suggestion! Updated.